languages/de-DE/TiddlerInfo.multids

```diff
@@ -1,21 +1,21 @@
 title: $:/language/TiddlerInfo/
 
 Advanced/Caption: Erweitert
-Advanced/PluginInfo/Empty/Hint: keine
+Advanced/PluginInfo/Empty/Hint: Keine
 Advanced/PluginInfo/Heading: Plugin Details
 Advanced/PluginInfo/Hint: Dieses Plugin enthält folgende Schatten-Tiddler:
-Advanced/ShadowInfo/Heading: Shatten Status
+Advanced/ShadowInfo/Heading: Schatten Status
 Advanced/ShadowInfo/NotShadow/Hint: Der Tiddler: <$link to=<<infoTiddler>>><$text text=<<infoTiddler>>/></$link> ist kein Schatten-Tiddler.
 Advanced/ShadowInfo/Shadow/Hint: Der Tiddler: <$link to=<<infoTiddler>>><$text text=<<infoTiddler>>/></$link> ist ein Schatten-Tiddler.
 Advanced/ShadowInfo/Shadow/Source: Er ist definiert im Plugin: <$link to=<<pluginTiddler>>><$text text=<<pluginTiddler>>/></$link>.
-Advanced/ShadowInfo/OverriddenShadow/Hint: Der originale Schatten-Tiddler wurde durch diesen Tiddler überschrieben. Wenn Sie diesen Tiddler löschen, wird der originale Schatten-Tiddler wieder aktiv. Vorher eventuell eine Sicherungskopie erstellen!
+Advanced/ShadowInfo/OverriddenShadow/Hint: Der originale Schatten-Tiddler wurde durch diesen Tiddler überschrieben. Wenn Sie diesen Tiddler löschen, wird der originale Schatten-Tiddler wieder aktiv. Erstellen Sie vorher eventuell eine Sicherungskopie!
 Fields/Caption: Felder
 List/Caption: Liste
 List/Empty: Dieser Tiddler hat kein "list" Feld.
 Listed/Caption: Gelistet
 Listed/Empty: Dieser Tiddler wird nicht von anderen Tiddlern gelistet.
 References/Caption: Referenzen
-References/Empty: Kein Tiddler linkt zu Diesem.
+References/Empty: Kein Tiddler linkt zu diesem Tiddler.
 Tagging/Caption: Tagging
```

Comment:
kommt das Wort "Tiddler" dann nicht zu oft vor?

Reply:
I personally think that for starters it's easier to understand what's meant if you refer to the object more often. In my opinion it's better to sacrifice "good german" in favor of easier understanding

Reply:
Ich bin für...

> Kein Tiddler linkt zu diesem.

No idea why *Diesem* was *Capitalized* before. It doesn't seem valid grammar. I imagine this was what bugged you more than anything.

Reply:
"Shatten Status" bugged me more. I'm fine with "Kein Tiddler linkt zu diesem" as I'm fine with "Kein Tiddler linkt zu diesem Tiddler" ... just a matter of taste